modules/041-linstor/templates/nodegroupconfiguration-check-drbd-version.yaml

```
@@ -32,6 +32,8 @@ spec:
     current_version="$(cat /proc/drbd | grep 'version:' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')"
     desired_version="{{ $.Values.linstor.internal.drbdVersion }}"
 
+    if [[ $current_version =~ ^8.* ]]; then rmmod drbd_transport_rdma drbd_transport_tcp drbd; exit 0; fi
```

It's better to use multiline syntax to make this condition more obvious :
```suggestion
    if [[ $current_version =~ ^8.* ]]; then
      rmmod drbd_transport_rdma drbd_transport_tcp drbd
      exit 0
    fi
```